Update for UNC-Charlotte tenure track faculty 9/1/2022

smudge1872 opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

smudge1872 committed:
Added changes reflecting UNC Charlotte faculty

Added Aidong Lu google scholar.
Deleted Ashit Talukder
Changed Amirmohammad Rooshenas affliation to University of Illinois at Chicago

smudge1872 committed:
Added UNC-Charlotte tenure track Faculty

Added Christian Kümmerle

smudge1872 committed:
Updated for UNC-Charlotte faculty 

Changed Dazhao Cheng affiliation to Wuhan University

smudge1872 committed:
Update for UNC-Charlotte Faculty

Changed Gabriel Terejanu affiliation to UNC-Charlotte

smudge1872 committed:
Updated for UNC-Charlotte faculty

Updated Kalpathi R. Subramanian website
Deleted Keh-Hsun Chen

smudge1872 committed:
Updated for UNC-Charlotte faculty

Added Liyue Fan

smudge1872 committed:
Update for UNC-Charlotte faculty

Updated Min Shin
Added Min C. Shin and Minwoo Lee

smudge1872 committed:
Updated UNC-Charlotte faculty

Changed Pedram Rooshenas affliation to UIC
Added Pu Wang 0001

smudge1872 committed:
Update for UNC-Charlotte faculty

Updated Samira Shaikh
Deleted Shaoting Zhang, affliated with a company PJ Lab
Added Siddharth Krishnan, Srijan Das

smudge1872 committed:
Updated for UNC-Charlotte Faculty

Added Taghi Mostafavi and Tyler N. Allen

smudge1872 committed:
Added M. Taghi Mostafavi

smudge1872 committed:
Update UNC-Charlotte faculty

Added Wenhao Luo, Wlodek W. Zadrozny
Updated Wenwen Dou and Wlodek Zadrozny

smudge1872 committed:
Updated for UNC-Charlotte faculty

Added Xiang Zhang 0012

smudge1872 committed:
Updated for UNC Charlotte faculty

Added Yonghong Yan 0001

smudge1872 committed:
Updated UNC-Charlotte faculty

Updated Zachary Wartell

emeryberger commented on csrankings-m.csv:
```diff
@@ -1986,14 +1987,15 @@ Miltos Petridis,Middlesex University,http://www.cs.mdx.ac.uk/people/miltos-petri
 Min Long,Boise State University,http://flash.uchicago.edu/~long,Ex_raoQAAAAJ
 Min Lu 0002,Shenzhen University,http://deardeer.github.io,xFAU798AAAAJ
-Min Shin,UNC - Charlotte,https://viscenter.uncc.edu/min-shin,T50sdQ8AAAAJ
+Min Shin,UNC - Charlotte,https://sites.google.com/a/uncc.edu/mcshin/,BVPAmrwAAAAJ
```
Not the correct DBLP entry, it appears. See https://dblp.org/pid/67/341.html.

smudge1872 replied:
Min Shin had 2 of 3 publications associated with the Min C. Shin at UNC - Charlotte. I requested dblp to correct this. I will delete.

emeryberger commented on csrankings-m.csv:
```diff
@@ -2086,6 +2088,7 @@ Minsoo Rhu,KAIST,https://sites.google.com/view/kaist-via,u_3ShigAAAAJ
 Minsoo Ryu,Hanyang University,http://rtcc.hanyang.ac.kr/rtccw/?page_id=27,NOSCHOLARPAGE
 Minsu Cho,POSTECH,http://cvlab.postech.ac.kr/~mcho,5TyoF5QAAAAJ
 Minsuk Kahng,Oregon State University,https://minsuk.com,vieI1GYAAAAJ
+Minwoo Lee,UNC - Charlotte,https://webpages.uncc.edu/mlee173/,awrLUJ4AAAAJ
```
That is a disambiguation entry. Please point to the correct DBLP name. https://dblp.org/pid/37/8786.html

smudge1872 replied:
Min Shin had 2 of 3 publications associated with the Min C. Shin at UNC - Charlotte. I requested dblp to correct this.  I will delete.

smudge1872 replied:
I deleted Minwoo Lee. I will ask dblp to create entry for him.

emeryberger commented on csrankings-w.csv:
```diff
@@ -453,7 +454,8 @@ Wissam Fawaz,Lebanese American University,http://soe.lau.edu.lb/ece/faculty-staf
 Witold Charatonik,University of Wroclaw,https://www.ii.uni.wroc.pl/~wch,mCyQ2RgAAAAJ
 Wlodek Drabent,IPI PAN,https://ipipan.waw.pl/instytut/pracownicy/wlodzimierz-drabent,bG-Ml1YAAAAJ
-Wlodek Zadrozny,UNC - Charlotte,https://cci.uncc.edu/directory/wlodek-zadrozny,PwdKImsAAAAJ
+Wlodek W. Zadrozny,UNC - Charlotte,https://sites.google.com/a/uncc.edu/wzadrozny/,PwdKImsAAAAJ
```
Please remove this entry and ask DBLP to add an alias for it.

smudge1872 replied:
I removed "Wlodek W. Zadrozny" and will ask dblp to create an alias for it.

smudge1872 committed:
Deleted Min Shin

Deleted Min Shin. Min Shin had 2 of 3 publications associated with the Min C. Shin at UNC - Charlotte. I requested dblp to correct this.

smudge1872 committed:
Deleted Minwoo Lee

Deleted "Minwoo Lee,UNC - Charlotte,https://webpages.uncc.edu/mlee173/,awrLUJ4AAAAJ" because dblp does not have have an author page for him yet.

smudge1872 committed:
Deleted Wlodek W. Zadrozny

Deleted "Wlodek W. Zadrozny,UNC - Charlotte,https://sites.google.com/a/uncc.edu/wzadrozny/,PwdKImsAAAAJ".  Will ask dblp to make alias for it.

smudge1872 committed:
Merge branch 'gh-pages' into gh-pages

emeryberger commented on csrankings-w.csv:
```diff
@@ -456,7 +456,6 @@ Wissam Fawaz,Lebanese American University,http://soe.lau.edu.lb/ece/faculty-staf
 Witold Charatonik,University of Wroclaw,https://www.ii.uni.wroc.pl/~wch,mCyQ2RgAAAAJ
 Wlodek Drabent,IPI PAN,https://ipipan.waw.pl/instytut/pracownicy/wlodzimierz-drabent,bG-Ml1YAAAAJ
-Wlodek W. Zadrozny,UNC - Charlotte,https://sites.google.com/a/uncc.edu/wzadrozny/,PwdKImsAAAAJ
 Wlodek Zadrozny,UNC - Charlotte,https://sites.google.com/a/uncc.edu/wzadrozny/,PwdKImsAAAAJ
```
Please remove this duplicate entry.

smudge1872 replied:
I thought "Wlodek W. Zodrozny" was already removed. I don't see it in my fork anymore.

smudge1872 committed:
Merge branch 'gh-pages' into gh-pages

smudge1872 committed:
Updated Ahmed Helmy to UNC-Charlotte

Changed Ahmed Helmy affiliation to UNC-Charlotte.

smudge1872 committed:
Merge branch 'emeryberger:gh-pages' into gh-pages

smudge1872 commented:
It seems that that this pull request has gone stale. I have made the changes requested. I will try a new pull-request.
